drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```diff
@@ -0,0 +1,611 @@
+{
+  "item": [
+    {
+      "text": "Client Data",
+      "type": "GROUP",
+      "link_id": "caseworker_exit_client_data",
+      "item": [
+        {
+          "text": "Assessment Date",
+          "type": "DATE",
+          "link_id": "caseworker_assessment_date",
+          "required": true,
+          "mapping": {
+            "field_name": "assessmentDate"
+          },
+          "assessment_date": true
+        },
+        {
+          "text": "Exit Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_date"
+          }
+        },
+        {
+          "text": "Phone Number",
+          "size": "SMALL",
+          "type": "STRING",
+          "link_id": "caseworker_exit_phone",
+          "component": "PHONE",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_phone"
+          }
+        },
+        {
+          "text": "Email",
+          "type": "STRING",
+          "link_id": "caseworker_exit_email",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_email"
+          }
+        },
+        {
+          "text": "Number of weeks in program",
+          "type": "INTEGER",
+          "required": true,
+          "link_id": "caseworker_exit_weeks_in_program",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_weeks_in_program"
+          }
+        },
+        {
+          "text": "Beginning GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "NUMBER",
+          "link_id": "caseworker_exit_beginning_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score"
+          }
+        },
+        {
+          "text": "Beginning GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_beginning_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score_date"
+          }
+        },
+        {
+          "text": "Exit GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_end_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score"
+          }
+        },
+        {
+          "text": "Exit GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_end_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score_date"
+          }
+        },
+        {
+          "text": "Difference/Change in GAS score",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_gas_score_change",
+          "read_only": true,
+          "autofill_values": [
+            {
+              "formula": "caseworker_exit_end_gas_score - caseworker_exit_beginning_gas_score",
+              "autofill_when": [
```

Comment:
I'm also seeing that `autofill_when` doesn't correctly update when a field is filled in but then deleted, as `enable_when` does. Same question here, should I debug and fix now or create a separate ticket?

Reply:
Good find, a separate ticket would be great. I'm not sure exactly how it should behave, that's a tricky one. Although in this case if you leave `autofill_when` empty, it is probably better, because it reacts to when one or both values are removed.

Reply:
Ok sounds good, I created https://github.com/open-path/Green-River/issues/5954 and will make that change to this form